frb_example/pure_dart/dart/lib/main.dart

```diff
@@ -600,6 +600,11 @@ void main(List<String> args) async {
     expect(n.field, 1);
   });
 
+  test('test empty struct', () async {
+    final empty = Empty();
+    await api.emptyStruct(empty: empty);
```

Comment:
uber-nit: maybe add a little check

```suggestion
    final output = await api.emptyStruct(empty: empty);
    expect(output, isA<Empty>());
```